"use strict";(self.webpackChunkrobokop_api_docs=self.webpackChunkrobokop_api_docs||[]).push([[6700],{66242:(e,t,n)=>{n.d(t,{Z:()=>b});var a=n(67294),o=n(86010),r=n(53438),i=n(39960),s=n(13919),l=n(95999);const c={cardContainer:"cardContainer_S8oU",cardTitle:"cardTitle_HoSo",cardDescription:"cardDescription_c27F",info:"info_QY_6"};var d=n(25108);function u(e){let{href:t,children:n}=e;return a.createElement(i.default,{href:t,className:(0,o.default)("card padding--lg",c.cardContainer)},n)}function m(e){let{href:t,icon:n,title:r,description:i,info:s}=e;return a.createElement(u,{href:t},a.createElement("h2",{className:(0,o.default)("text--truncate",c.cardTitle),title:r},n," ",r),i&&a.createElement("p",{className:(0,o.default)("text--truncate",c.cardDescription),title:i},i),s&&a.createElement("p",{title:s,className:c.info},s))}const p=new Map([["BINDING","binding-db"],["CAM Provider KG","cam-kp"],["CTD","ctd"],["DrugCentral","drugcentral"],["Alliance of Genome Resources","genome-alliance"],["GTEx","gtex"],["GtoPdb","gtopdb"],["GWAS Catalog","gwas-catalog"],["Hetionet","hetio"],["HGNC","hgnc"],["HMDB","hmdb"],["Human GOA","human-goa"],["ICEES KG","icees-kg"],["IntAct Molecular Interaction Database","intact"],["Monarch KG","monarch-kg"],["PANTHER","panther"],["Pharos","pharos"],["Reactome","reactome"],["ROBOKOP KG","robokopkg"],["STRING","string-db"],["Text Mining Provider KG","textminingkp"],["Ubergraph","ubergraph"],["Viral Proteome","viral-proteome"]]),g=async e=>{if(!p.has(e))return null;try{const t=await fetch(`https://robokop-automat.apps.renci.org/${p.get(e)}/metadata`,{method:"GET",headers:{"Content-Type":"application/json"}}),{final_node_count:n,final_edge_count:a,graph_description:o}=await t.json();return{final_node_count:parseInt(n).toLocaleString(),final_edge_count:parseInt(a).toLocaleString(),description:o}}catch{return null}};function f(e){let{item:t}=e;const n=(0,r.Wl)(t),[o,i]=(0,a.useState)(null);return(0,a.useEffect)((()=>{(async()=>{i(await g(t.label))})()}),[]),d.log(t),n?a.createElement(m,{href:n,icon:"\ud83d\uddc3\ufe0f",title:t.label,description:(0,l.I)({message:o&&"NaN"!==o.final_edge_count&&"NaN"!==o.final_node_count?"{final_node_count} nodes, {final_edge_count} edges":"{count} items",id:"theme.docs.DocCard.categoryDescription",description:"The default description for a category card in the generated index about how many items this category includes"},{count:t.items.length,final_node_count:null==o?void 0:o.final_node_count,final_edge_count:null==o?void 0:o.final_edge_count}),info:null==o?void 0:o.description}):null}function h(e){let{item:t}=e;const n=(0,s.Z)(t.href)?"\ud83d\udcc4\ufe0f":"\ud83d\udd17",o=(0,r.xz)(t.docId??void 0);return a.createElement(m,{href:t.href,icon:n,title:t.label,description:null==o?void 0:o.description})}function _(e){let{item:t}=e;switch(t.type){case"link":return a.createElement(h,{item:t});case"category":return a.createElement(f,{item:t});default:throw new Error(`unknown item type ${JSON.stringify(t)}`)}}function k(e){let{className:t}=e;const n=(0,r.jA)();return a.createElement(b,{items:n.items,className:t})}function b(e){const{items:t,className:n}=e;if(!t)return a.createElement(k,e);const i=(0,r.MN)(t);return a.createElement("section",{className:(0,o.default)("row",n)},i.map(((e,t)=>a.createElement("article",{key:t,className:"col col--6 margin-bottom--lg"},a.createElement(_,{item:e})))))}},13704:(e,t,n)=>{n.r(t),n.d(t,{assets:()=>d,contentTitle:()=>l,default:()=>g,frontMatter:()=>s,metadata:()=>c,toc:()=>u});var a=n(87462),o=(n(67294),n(3905)),r=n(66242),i=n(53438);const s={id:"alliance-of-genome-resources",title:"Alliance of Genome Resources",description:"Alliance of Genome Resources",custom_edit_url:null},l=void 0,c={unversionedId:"automat/alliance-of-genome-resources",id:"automat/alliance-of-genome-resources",title:"Alliance of Genome Resources",description:"Alliance of Genome Resources",source:"@site/docs/automat/alliance-of-genome-resources.tag.mdx",sourceDirName:"automat",slug:"/automat/alliance-of-genome-resources",permalink:"/api-docs/docs/automat/alliance-of-genome-resources",draft:!1,editUrl:null,tags:[],version:"current",frontMatter:{id:"alliance-of-genome-resources",title:"Alliance of Genome Resources",description:"Alliance of Genome Resources",custom_edit_url:null},sidebar:"openApiSidebar",previous:{title:"Test data for usage by the SRI Testing Harness.",permalink:"/api-docs/docs/automat/get-sri-testing-data-1-4-sri-testing-data-get-drugcentral-trapi"},next:{title:"Query reasoner via one of several inputs.",permalink:"/api-docs/docs/automat/reasoner-api-1-4-query-post-genome-alliance-trapi"}},d={},u=[{value:"Query Endpoints",id:"query-endpoints",level:2},{value:"Metadata Endpoints",id:"metadata-endpoints",level:2}],m={toc:u},p="wrapper";function g(e){let{components:t,...n}=e;return(0,o.kt)(p,(0,a.Z)({},m,n,{components:t,mdxType:"MDXLayout"}),(0,o.kt)("p",null,"The Alliance of Genome Resources is a consortium of the owners of seven model organism databases (MODs) and the Gene Ontology (GO) Consortium, whose goal is to provide an integrated view of their data to all biologists, clinicians, and other interested parties. The Alliance provides the results of all methods that have been benchmarked by the Quest for Orthologs Consortium (QfO), as well as curated ortholog inferences from HGNC (human, mouse, rat genes)."),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"Version:")," 52b222baedc80328"),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"URL:")," ",(0,o.kt)("a",{parentName:"p",href:"https://www.alliancegenome.org/"},"https://www.alliancegenome.org/")),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"Files:")," ",(0,o.kt)("a",{parentName:"p",href:"https://stars.renci.org/var/plater/bl-3.5.4/GenomeAllianceOrthologs_Automat/52b222baedc80328"},"https://stars.renci.org/var/plater/bl-3.5.4/GenomeAllianceOrthologs_Automat/52b222baedc80328")),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"Nodes:")," 94,336"),(0,o.kt)("p",null,(0,o.kt)("strong",{parentName:"p"},"Edges:")," 530,374"),(0,o.kt)("h2",{id:"query-endpoints"},"Query Endpoints"),(0,o.kt)(r.Z,{items:(0,i.jA)().items.slice(0,6),mdxType:"DocCardList"}),(0,o.kt)("h2",{id:"metadata-endpoints"},"Metadata Endpoints"),(0,o.kt)(r.Z,{items:(0,i.jA)().items.slice(6),mdxType:"DocCardList"}))}g.isMDXComponent=!0}}]);