"use strict";(self.webpackChunkrobokop_api_docs=self.webpackChunkrobokop_api_docs||[]).push([[2503],{66242:(e,t,o)=>{o.d(t,{Z:()=>b});var n=o(67294),a=o(86010),r=o(53438),i=o(39960),l=o(13919),c=o(95999);const s={cardContainer:"cardContainer_S8oU",cardTitle:"cardTitle_HoSo",cardDescription:"cardDescription_c27F",info:"info_QY_6"};function m(e){let{href:t,children:o}=e;return n.createElement(i.Z,{href:t,className:(0,a.Z)("card padding--lg",s.cardContainer)},o)}function d(e){let{href:t,icon:o,title:r,description:i,info:l}=e;return n.createElement(m,{href:t},n.createElement("h2",{className:(0,a.Z)("text--truncate",s.cardTitle),title:r},o," ",r),i&&n.createElement("p",{className:(0,a.Z)("text--truncate",s.cardDescription),title:i},i),l&&n.createElement("p",{title:l,className:s.info},l))}const u=new Map([["RENCI SRI Reference KG","biolink"],["CAM KP","cam-kp"],["CTD","ctd"],["DrugCentral","drugcentral"],["Alliance of Genome Resources Orthology","genome-alliance"],["GTEx","gtex"],["GtoPdb","gtopdb"],["GWAS Catalog","gwas-catalog"],["Hetionet","hetio"],["HGNC","hgnc"],["HMDB","hmdb"],["Human GOA","human-goa"],["ICEES KG","icees-kg"],["IntAct Molecular Interaction Database","intact"],["PANTHER","panther"],["Pharos","pharos"],["ROBOKOP KG","robokopkg"],["SRI Reference KG","sri-reference-kg"],["STRING","string-db"],["Text Mining Provider KG","textminingkp"],["Ubergraph","ubergraph"],["Viral Proteome","viral-proteome"]]),p=async e=>{if(!u.has(e))return null;try{const t=await fetch(`https://automat-u24.apps.renci.org/${u.get(e)}/metadata`,{method:"GET",headers:{"Content-Type":"application/json"}}),{final_node_count:o,final_edge_count:n,graph_description:a}=await t.json();return{final_node_count:parseInt(o).toLocaleString(),final_edge_count:parseInt(n).toLocaleString(),description:a}}catch{return null}};function g(e){let{item:t}=e;const o=(0,r.Wl)(t),[a,i]=(0,n.useState)(null);return(0,n.useEffect)((()=>{(async()=>{i(await p(t.label))})()}),[]),o?n.createElement(d,{href:o,icon:"\ud83d\uddc3\ufe0f",title:t.label,description:(0,c.I)({message:a&&"NaN"!==a.final_edge_count&&"NaN"!==a.final_node_count?"{final_node_count} nodes, {final_edge_count} edges":"{count} items",id:"theme.docs.DocCard.categoryDescription",description:"The default description for a category card in the generated index about how many items this category includes"},{count:t.items.length,final_node_count:a?.final_node_count,final_edge_count:a?.final_edge_count}),info:a?.description}):null}function f(e){let{item:t}=e;const o=(0,l.Z)(t.href)?"\ud83d\udcc4\ufe0f":"\ud83d\udd17",a=(0,r.xz)(t.docId??void 0);return n.createElement(d,{href:t.href,icon:o,title:t.label,description:a?.description})}function h(e){let{item:t}=e;switch(t.type){case"link":return n.createElement(f,{item:t});case"category":return n.createElement(g,{item:t});default:throw new Error(`unknown item type ${JSON.stringify(t)}`)}}function _(e){let{className:t}=e;const o=(0,r.jA)();return n.createElement(b,{items:o.items,className:t})}function b(e){const{items:t,className:o}=e;if(!t)return n.createElement(_,e);const i=(0,r.MN)(t);return n.createElement("section",{className:(0,a.Z)("row",o)},i.map(((e,t)=>n.createElement("article",{key:t,className:"col col--6 margin-bottom--lg"},n.createElement(h,{item:e})))))}},97955:(e,t,o)=>{o.r(t),o.d(t,{assets:()=>m,contentTitle:()=>c,default:()=>g,frontMatter:()=>l,metadata:()=>s,toc:()=>d});var n=o(87462),a=(o(67294),o(3905)),r=o(66242),i=o(53438);const l={id:"alliance-of-genome-resources-orthology",title:"Alliance of Genome Resources Orthology",description:"Alliance of Genome Resources Orthology",custom_edit_url:null},c=void 0,s={unversionedId:"automat/alliance-of-genome-resources-orthology",id:"automat/alliance-of-genome-resources-orthology",title:"Alliance of Genome Resources Orthology",description:"Alliance of Genome Resources Orthology",source:"@site/docs/automat/alliance-of-genome-resources-orthology.tag.mdx",sourceDirName:"automat",slug:"/automat/alliance-of-genome-resources-orthology",permalink:"/api-docs/docs/automat/alliance-of-genome-resources-orthology",draft:!1,editUrl:null,tags:[],version:"current",frontMatter:{id:"alliance-of-genome-resources-orthology",title:"Alliance of Genome Resources Orthology",description:"Alliance of Genome Resources Orthology",custom_edit_url:null},sidebar:"openApiSidebar",previous:{title:"Get one-hop connection schema",permalink:"/api-docs/docs/automat/simple-spec-simple-spec-get-drugcentral"},next:{title:"Meta knowledge graph representation of this TRAPI web service.",permalink:"/api-docs/docs/automat/get-meta-knowledge-graph-1-3-meta-knowledge-graph-get-genome-alliance-trapi"}},m={},d=[],u={toc:d},p="wrapper";function g(e){let{components:t,...o}=e;return(0,a.kt)(p,(0,n.Z)({},u,o,{components:t,mdxType:"MDXLayout"}),(0,a.kt)("p",null,"The Alliance of Genome Resources is a consortium of 7 model organism databases (MODs) and the Gene Ontology (GO) Consortium whose goal is to provide an integrated view of their data to all biologists, clinicians and other interested parties. The Alliance provides the results of all methods that have been benchmarked by the Quest for Orthologs Consortium (QfO), as well as curated ortholog inferences from HGNC (for human, mouse and rat genes)."),(0,a.kt)("p",null,(0,a.kt)("strong",{parentName:"p"},"Version:")," eeadb0cfeb83a973"),(0,a.kt)("p",null,(0,a.kt)("strong",{parentName:"p"},"URL:")," ",(0,a.kt)("a",{parentName:"p",href:"https://www.alliancegenome.org/"},"https://www.alliancegenome.org/")),(0,a.kt)("p",null,(0,a.kt)("strong",{parentName:"p"},"Neo4J:")," ",(0,a.kt)("a",{parentName:"p",href:"https://stars.renci.org/var/plater/bl-3.2.1/GenomeAllianceOrthologs_Automat/eeadb0cfeb83a973/graph_eeadb0cfeb83a973.db.dump"},"https://stars.renci.org/var/plater/bl-3.2.1/GenomeAllianceOrthologs_Automat/eeadb0cfeb83a973/graph_eeadb0cfeb83a973.db.dump")),(0,a.kt)("p",null,(0,a.kt)("strong",{parentName:"p"},"Nodes:")," 94,352"),(0,a.kt)("p",null,(0,a.kt)("strong",{parentName:"p"},"Edges:")," 530,513"),(0,a.kt)(r.Z,{items:(0,i.jA)().items,mdxType:"DocCardList"}))}g.isMDXComponent=!0}}]);